build/build.rb

```diff
@@ -89,6 +90,8 @@ def verify_run(output)
     file_name = "#{name}-#{target}-#{build_label}-#{timestamp}-#{git_checkout.commit_sha}.#{FILE_TYPE[imgfac_target]}"
     destination = DESTINATION_DIRECTORY.join(file_name)
     $log.info `mv  #{source} #{destination}`
+    $log.info `cd #{DESTINATION_DIRECTORY}/; /usr/bin/md5sum * >> md5checksums\""`
+    $log.info `cd #{DESTINATION_DIRECTORY}/; /usr/bin/sha256sum * >> 256checksums\""`
```

Comment:
If you don't want the md2checksums file to be included in the 256checksums file you can break it apart:

```ruby
$log.info "Generating image checksums"
md5sums = `cd #{DESTINATION_DIRECTORY}/; /usr/bin/md5sum *`
256sums = `cd #{DESTINATION_DIRECTORY}/; /usr/bin/sha256sum *`
File.write(File.join(DESTINATION_DIRECTORY, "md5checksums"), md5sums)
File.write(File.join(DESTINATION_DIRECTORY, "256checksums"), 256sums)
```